### ISSUE TITLE
Support For Indigenous Languages | Spike

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#1505]
### Acceptance Criteria
- Where to get Indigenous names we can use to test?
- investigate the use of indigenous languages
-

**Comment by jleach** [comment]
@cvarjao For testing I've installed a Salish keyboard and enter some words and phrases from language tutorials. This might be a good place to start. Make sure we can issue, verify and text message words with a reasonably common language to BC.

**Comment by knguyenBC** [comment]
Needs investigation on indigenous language and use in bc services.

**Comment by jeznorth** [comment]
@knguyenBC can you check who will be responsible for this?

**Comment by jeznorth** [comment]
Given we are leveraging the 'BC Sans' typeface, would it be safe to assume we have inherent support?

**Comment by jleach** [comment]
@jeznorth Yes, that was the intent. We just don't have any translations for it.
